src/components/button-row/button-row.js

```diff
@@ -47,9 +32,33 @@ class ButtonRow extends React.Component {
     return components;
   }
 
+  /**
+  Creates the inline list based on the provided props.
+  @method _generateButtonsList
+  @param {Array} components The properties of the Button components to generate.
+  @returns {Array} Collection of inline list items.
+*/
+  _generateButtonsList(components) {
+    var listItems = [];
+    components.forEach((component, index) => {
```

Comment:
Because you're creating a new array for every element `map` is a more appropriate method here.